### PR TITLE
Using skywalking goapi as protocol

### PR DESCRIPTION
**Pull request by mrproliu**
Using [SkyWalking GoAPI](https://github.com/apache/skywalking-goapi) as protocol, remove self-build protocol files

### CODE REVIEW

File: noop.go [hunk 1]
```diff
@@ -20,7 +20,7 @@ package go2sky
 import (
 	"time"
 
-	v3 "github.com/SkyAPM/go2sky/reporter/grpc/language-agent"
+	v3 "skywalking.apache.org/repo/goapi/collect/language/agent/v3"
```
Comment (kezhenxu94) [inline noop.go:23]: Redundant alias

File: noop_test.go [hunk 2]
```diff
@@ -22,7 +22,7 @@ import (
 	"testing"
 	"time"
 
-	v3 "github.com/SkyAPM/go2sky/reporter/grpc/language-agent"
+	v3 "skywalking.apache.org/repo/goapi/collect/language/agent/v3"
```
Comment (kezhenxu94) [inline noop_test.go:25]: Redundant alias

File: plugins/http/client.go [hunk 3]
```diff
@@ -23,7 +23,7 @@ import (
 	"time"
 
 	"github.com/SkyAPM/go2sky"
-	v3 "github.com/SkyAPM/go2sky/reporter/grpc/language-agent"
+	v3 "skywalking.apache.org/repo/goapi/collect/language/agent/v3"
```
Comment (kezhenxu94) [inline plugins/http/client.go:26]: Redundant alias

File: plugins/http/server.go [hunk 4]
```diff
@@ -25,7 +25,7 @@ import (
 
 	"github.com/SkyAPM/go2sky"
 	"github.com/SkyAPM/go2sky/internal/tool"
-	v3 "github.com/SkyAPM/go2sky/reporter/grpc/language-agent"
+	v3 "skywalking.apache.org/repo/goapi/collect/language/agent/v3"
```
Comment (kezhenxu94) [inline plugins/http/server.go:28]: Redundant alias

File: span_test.go [hunk 5]
```diff
@@ -22,7 +22,7 @@ import (
 	"testing"
 	"time"
 
-	v3 "github.com/SkyAPM/go2sky/reporter/grpc/language-agent"
+	v3 "skywalking.apache.org/repo/goapi/collect/language/agent/v3"
```
Comment (kezhenxu94) [inline span_test.go:25]: Redundant alias

**Review by kezhenxu94 — APPROVED** [review]
Good

**Comment by mrproliu** [comment]
Hi, @kezhenxu94. The redundant aliases are generated by `goimports` plugin. If I remove it, it will throw an error in the lint phase.

**Comment by kezhenxu94** [comment]
> Hi, @kezhenxu94. The redundant aliases are generated by `goimports` plugin. If I remove it, it will throw an error in the lint phase.

That’s weird, anyway, it looks good to me

File: noop.go (hunk 1 above)
Comment (wu-sheng) [reply noop.go:23]: I tried to remove `v3` alias, nothing wrong happens. @mrproliu Could you recheck what happens?

**Review by wu-sheng — CHANGES_REQUESTED** [review]
Alias needs to recheck.

File: noop.go (hunk 1 above)
Comment (mrproliu) [reply noop.go:23]: image

Hi, here is the error that I got. It notices me the files are not `goimports`, is there did I use it wrong?

Comment (wu-sheng) [reply noop.go:23]: Seems your golangci-lint issue. I can see from my logs, everything is fine.

Comment (wu-sheng) [reply noop.go:23]: You could try to submit a version without alias, and let CI to recheck my result for you.

Comment (mrproliu) [reply noop.go:23]: I have to try to use `goimport` command on the `noop.go` files. There is the command `goimports noop.go`, and it output the same result(include the `v3` alias), is I miss anything? Also, I found [an issue](https://github.com/golang/go/issues/30051), it seems like the redundant alias is tight. 
image

**Comment by kezhenxu94** [comment]
After reading the issues, I think the aliases are inevitable, but I'd suggest use `agentv3` as alias for `"skywalking.apache.org/repo/goapi/collect/language/agent/v3"`

**Comment by wu-sheng** [comment]
> After reading the issues, I think the aliases are inevitable

Could you explain a little? @kezhenxu94

**Comment by wu-sheng** [comment]
> > After reading the issues, I think the aliases are inevitable
> 
> Could you explain a little? @kezhenxu94

OK, I can see this is useful in the `client.go` as it needs `v3.SpanLayer_Http`.  @mrproliu Please follow the renaming suggestion.

File: noop_test.go (hunk 2 above)
Comment (wu-sheng) [inline noop_test.go:25]: Miss this?

Comment (mrproliu) [reply noop_test.go]: Fixed

**Review by wu-sheng — APPROVED** [review]

**Pull request merged**